Firestore/Source/Remote/FSTStream.mm

```diff
@@ -550,11 +546,7 @@ - (void)handleStreamClose:(nullable NSError *)error {
  */
 - (void)writeValue:(id)value __used {
   [self.workerDispatchQueue verifyIsCurrentQueue];
-
-  if (![self isStarted]) {
-    FSTLog(@"%@ Ignoring stream message from inactive stream.", NSStringFromClass([self class]));
-    return;
-  }
+  FSTAssert([self isStarted], @"writeValue: called for stopped stream.");
```

Comment:
In contrast to the above, this and below should be safe again because they're only called by gRPC.